applications/ALEapplication/python_scripts/ale_navier_stokes_solver_fractionalstep.py

```diff
@@ -20,7 +20,7 @@ def CreateSolver(model_part, custom_settings):
     return ALENavierStokesSolverFractionalStep(model_part, custom_settings)
 
 
-class ALENavierStokesSolverFractionalStep(navier_stokes_solver_fractionalstep.NavierStokesSolver_FractionalStep):
+class ALENavierStokesSolverFractionalStep(navier_stokes_solver_fractionalstep.NavierStokesSolverFractionalStep):
```

Comment:
this is quite off topic, however it would be ideal if we defined (and used in Solve) a subdivision in
 Predict, 
IntializeSolutionStep 
SolveSolutionStep
FinalizeSolutionStep

for all of such solvers. This would make them compatible with the analysis...

Reply:
Can do, thx for the reminder

WIll do it together with the update of the MeshMovingAnalysis (AnalysisStage)

Reply:
as another comment, again offtopic, but since you couple the solver and the fluid solver you could ensure that the scheme employed to compute mesh velocity is the same (i mean BDF with BDF, newmark with newmark...)